viewing_party/party.py

```diff
@@ -1,23 +1,152 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    new_movie = {}
+    if not title or not genre or not rating:
+        return None
+    new_movie["title"] = title
+    new_movie["genre"] = genre
+    new_movie["rating"] = rating
+    return new_movie
+
+
+def add_to_watched(user_data, movie):
+    movies_watched_list = user_data["watched"]
+    movies_watched_list.append(movie)

```

Comment:
You can simplify this by appending to the `user_data["watched"]` instead of create a new variable.